lib/dfe/reference_data/record.rb

```diff
@@ -23,6 +23,14 @@ def ==(other)
                   other.data
                 end
       end
+
+      def to_s
+        if data.key?(:name)
+          "#<DfE::ReferenceData::Record id=#{id} name=#{name}>"
+        else
+          "#<DfE::ReferenceData::Record id=#{id}>"
+        end
+      end
```

Comment:
I didn't understand the reason of this method. What is the purpose to overwrite the to_s?

Reply:
If you overwrite and the record has many properties, those properties won't be show, is that expected?

Reply:
Oh, I just did that because the full descriptions of the records were making debugging output too voluminous to observe - as the records aren't really _modified_ by the code, as opposed to big lists of them being modified, I found that just displaying the ID and name let me know which records were were, which was what mattered. The rest of the contents of the record can be found by looking up the ID in the source file, if needed!

Reply:
I see, I'd recommend overwriting the #inspect then (or using the awesome print gem: https://github.com/awesome-print/awesome_print)